ERCS/erc-dvp.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
```suggestion
---
eip: 7573
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
I have shortened the abstract and moved details to later sections.